### PR TITLE
Try crashing this shit now

### PR DESCRIPTION
**Pull request by kibartas**
Fix crashing. Probably.
> Internet connection required to submit images.
> Fix quickly pressing start webcam and submit simultaneously crashing program.

### CODE REVIEW

File: EmotionCalculator/EmotionCalculator/FormsUI/BaseForm.cs [hunk 1]
```diff
@@ -125,12 +130,22 @@ private void CameraStopButton_Click(object sender, EventArgs e)
 
         private async void SubmitWebCamButton_Click(object sender, EventArgs e)
         {
-            Image image = null;
+            if (!ConnectionChecker.IsConnectedToInternet())
```
Comment (Julius-Syvis) [inline EmotionCalculator/EmotionCalculator/FormsUI/BaseForm.cs:133]: ok tai siūlau error išmest useriui "internet connection is required" ar sth

**Review by Julius-Syvis — CHANGES_REQUESTED** [review]

File: EmotionCalculator/EmotionCalculator/FormsUI/BaseForm.cs (hunk 1 above)
Comment (kibartas) [reply EmotionCalculator/EmotionCalculator/FormsUI/BaseForm.cs:133]: Can do that

File: EmotionCalculator/EmotionCalculator/Tools/FileHandler/ImageHandle.cs [hunk 2]
```diff
@@ -11,8 +12,10 @@ class ImageHandle
 
         public ImageHandle()
         {
-            file = new OpenFileDialog();
-            file.Filter = "JPG(*.JPG)|*.jpg";
+            file = new OpenFileDialog
```
Comment (Mantulis00) [inline EmotionCalculator/EmotionCalculator/Tools/FileHandler/ImageHandle.cs:15]: Tipo kažkoks skirtumas yra ?

**Review by Mantulis00 — COMMENTED** [review]
Pora vietų, manau, funkcionalumas nesiskiria, bet atrodo viskas OK

**Comment by Mantulis00** [comment]
Nebereikalingas mergas

**Pull request closed without merging**